plugins/modules/monit.py

```diff
@@ -154,35 +169,38 @@ def get_status(self, validate=False):
         rc, out, err = self.module.run_command(command, check_rc=check_rc)
         return self._parse_status(out, err)
 
-    def _parse_status(self, output, err):
+    def _parse_status(self, output, err) -> Status:
         escaped_monit_services = "|".join([re.escape(x) for x in MONIT_SERVICES])
         pattern = f"({escaped_monit_services}) '{re.escape(self.process_name)}'"
         if not re.search(pattern, output, re.IGNORECASE):
-            return Status.MISSING
+            return Status("MISSING")
 
-        status_val = re.findall(r"^\s*status\s*([\w\- ]+)", output, re.MULTILINE)
-        if not status_val:
+        status_val_find = re.findall(r"^\s*status\s*([\w\- ]+)", output, re.MULTILINE)
+        if not status_val_find:
             self.exit_fail("Unable to find process status", stdout=output, stderr=err)
 
-        status_val = status_val[0].strip().upper()
+        status_val = status_val_find[0].strip().upper()
         if " | " in status_val:
             status_val = status_val.split(" | ")[0]
         if " - " not in status_val:
             status_val = status_val.replace(" ", "_")
+            # Normalize RUNNING to OK (monit reports both, they mean the same thing)
+            if status_val == "RUNNING":
+                status_val = "OK"
             try:
-                return getattr(Status, status_val)
+                return Status(status_val)
             except AttributeError:
                 self.module.warn(f"Unknown monit status '{status_val}', treating as execution failed")
-                return Status.EXECUTION_FAILED
+                return Status("EXECUTION_FAILED")
         else:
             status_val, substatus = status_val.split(" - ")
             action, state = substatus.split()
             if action in ["START", "INITIALIZING", "RESTART", "MONITOR"]:
-                status = Status.OK
+                status = Status("OK")
             else:
-                status = Status.NOT_MONITORED
+                status = Status("NOT_MONITORED")
 
-            if state == "pending":
+            if state == "PENDING":
```

Comment:
Looks like this is a bugfix - I had not noticed it before. Because `status_val` is all uppercase (from line 182), when `state` is assigned to, the value cannot be lowercase.